syntax = "proto2";

package certain;

option cc_generic_services = true;

message DumpEntryReq {
  optional uint64 entity_id = 1;
  optional uint64 entry = 2;
}

message DumpEntryRsp {}

service ToolsService {
  //
  rpc DumpEntry(DumpEntryReq) returns (DumpEntryRsp);
}
